chroma_api/task.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2020 DDN. All rights reserved.
```

Comment:
We should not build any more api endpoints in Python.

Reasons:

- We are using a lib that is loosely maintained and is dependant on runtime RPMs to function.
- These RPMs are subject to change.
- The Python API has significantly worse performance characteristics.
- We have an existing Rust service that can host API endpoints, and communicate with the job_scheduler / db.

At this point, we should consider the Python API legacy, and move to replace it instead of building on top of it.